src/algebra/order/with_zero.lean

```diff
@@ -285,15 +285,36 @@ end
 lemma mul_le_mul_right₀ (hc : c ≠ 0) : a * c ≤ b * c ↔ a ≤ b :=
 ⟨le_of_le_mul_right hc, λ hab, mul_le_mul_right' hab _⟩
 
+lemma mul_le_mul_left₀ (ha : a ≠ 0) : a * b ≤ a * c ↔ b ≤ c :=
+by {simp only [mul_comm a], exact mul_le_mul_right₀ ha }
+
 lemma div_le_div_right₀ (hc : c ≠ 0) : a/c ≤ b/c ↔ a ≤ b :=
 by rw [div_eq_mul_inv, div_eq_mul_inv, mul_le_mul_right₀ (inv_ne_zero hc)]
 
+lemma div_le_div_left₀ (ha : a ≠ 0) (hb : b ≠ 0) (hc : c ≠ 0) : a/b ≤ a/c ↔ c ≤ b :=
+by simp only [div_eq_mul_inv, mul_le_mul_left₀ ha, inv_le_inv₀ hb hc]
+
 lemma le_div_iff₀ (hc : c ≠ 0) : a ≤ b/c ↔ a*c ≤ b :=
 by rw [div_eq_mul_inv, le_mul_inv_iff₀ hc]
 
 lemma div_le_iff₀ (hc : c ≠ 0) : a/c ≤ b ↔ a ≤ b*c :=
 by rw [div_eq_mul_inv, mul_inv_le_iff₀ hc]
 
+
+/-- `equiv.mul_left₀` as an order_iso.
+
+Note that `order_iso.mul_left₀` refers to the `linear_ordered_field` version. -/
+@[simps {simp_rhs := tt}]
```

Comment:
AFAIR, `simps` generate bad lemmas for `order_iso`s: it uses wrong `symm`. You can use `simps` to generate `apply` and `to_equiv`, then explicitly add a lemma
```
lemma order_iso.mul_left₀'_symm {a : α} (ha : a ≠ 0) :
  (order_iso.mul_left₀' a ha).symm = order_iso.mul_left₀' a⁻¹ (inv_ne_zero ha) :=
by { ext, refl }
```

Reply:
This lemma should be refl, right? No need for extensionality.

Reply:
Does this apply to [order_iso.mul_left₀](https://leanprover-community.github.io/mathlib_docs/algebra/order/field.html#order_iso.mul_left%E2%82%80) (the `linear_ordered_field` version) as well?

Reply:
Yael, `refl` alone didn't work for that lemma.